### PR TITLE
3.1 Fix missing_view template

### PR DESCRIPTION
**Pull request by antograssiot**

### CODE REVIEW

File: src/View/ViewBuilder.php [hunk 1]
```diff
@@ -322,7 +322,7 @@ public function build($vars = [], Request $request = null, Response $response =
             $className = App::className($className, 'View', 'View');
         }
         if (!$className) {
-            throw new MissingViewException([$this->_className]);
+            throw new MissingViewException(['class' => $this->_className]);
```
Comment (ADmad) [inline src/View/ViewBuilder.php]: We should probably also rename the exception class to `MissingTemplateException`.

Comment (antograssiot) [inline src/View/ViewBuilder.php]: @ADmad I'm not following, MissingTemplateException already exist but in this case it's actually a missing view class I believe, right ?

**Comment by lorenzo** [comment]
Thanks!

File: src/View/ViewBuilder.php (hunk 1 above)
Comment (ADmad) [inline src/View/ViewBuilder.php]: Sorry, I was mistaken.

**Pull request merged**